src/game/default/g_entity_target.c

```diff
@@ -195,7 +195,7 @@ void G_target_speaker(g_entity_t *ent) {
 		ent->s.sound = ent->locals.noise_index;
 	}
 
-	ent->locals.Use = (void *) &G_target_speaker_Use;
+	ent->locals.Use = G_target_speaker_Use;
```

Comment:
No idea :) `git blame` shows @tapir introduced that cast -- my guess is to fix a compiler warning on his end. But the way you have it now is the most "correct."